### PR TITLE
Add a recipe for pyremap

### PR DESCRIPTION
**Pull request by xylar**
Checklist

- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vend other packages
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by xylar** [comment]
@conda-forge/help-python, could someone review this please?

**Comment by xylar** [comment]
Thanks very much, @mariusvniekerk

File: recipes/pyremap/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+  version: {{ version }}
+
+source:
+  url: https://github.com/xylar/{{ name }}/archive/{{ version }}.tar.gz
+  sha256: 3ca8da6fdefaf23f14a4556271e025aef0ab5480a0cae5da1cc3fa1bb46f82f3
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+  skip: true  # [win]
```
Comment (chrisburr) [inline recipes/pyremap/meta.yaml]: Can this be noarch? If not the python version should be constrained with skip instead

File: recipes/pyremap/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+
+requirements:
+  host:
+    - python >=3.6
+    - pip
+  run:
+    - python >=3.6
+    - numpy
+    - scipy
+    - netCDF4
```
Comment (chrisburr) [inline recipes/pyremap/meta.yaml]: Conda package names are always lower case

File: recipes/pyremap/meta.yaml (hunk 1 above)
Comment (xylar) [reply recipes/pyremap/meta.yaml]: Sorry about that.  I had it as noarch before and realized it doesn't work under Windows.  I have no desire to make it Windows compatible.  I'll change the recipe to skip `py<36` as well and take out the python constraint.

File: recipes/pyremap/meta.yaml (hunk 2 above)
Comment (xylar) [reply recipes/pyremap/meta.yaml]: Oops, sorry about that.  I'll need to fix that here and I think in some other places.

**Comment by xylar** [comment]
@chrisburr and @mariusvniekerk, I was hoping to fix the issues pointed out above in the feedstock itself but it looks like something got messed up in setting up the feedstock.  There's a repo but it's empty.  Should I just be patient?
https://github.com/conda-forge/pyremap-feedstock

**Comment by xylar** [comment]
I see on Giter that this is being worked on.

**Pull request merged**